egs2/minds14/asr1/conf/slurm-2.conf

```diff
@@ -0,0 +1,14 @@
+# Default configuration
+command sbatch --export=PATH
+option name=* --job-name $0
+option time=* --time $0
+option mem=* --mem-per-cpu $0
+option mem=0
+option num_threads=* --cpus-per-task $0
+option num_threads=1 --cpus-per-task 1
+option num_nodes=* --nodes $0
+default gpu=0
+option gpu=0 -p RM-shared
+option gpu=* -p GPU-shared --gres=gpu:v100-16:$0 # Recommend allocating more CPU than, or equal to the number of GPU
```

Comment:
the slurm need to be exactly the same as the template, please copy that from the `egs2/TEMPLATE`